src/vs/vscode.proposed.d.ts

```diff
@@ -1127,4 +1127,13 @@ declare module 'vscode' {
 
 	}
 	//#endregion
+
+	//#region Tasks
+	export interface TaskPresentationOptions {
+		/**
+		 * Controls whether the task is executed in a specific terminal group using split panes.
+		 */
+		terminalGroup?: string;
```

Comment:
I'm leaning more towards calling this `group`. That way we aren't tied to any specific presenter.